src/pytorch_lightning/utilities/imports.py

```diff
@@ -23,6 +23,7 @@
 _TORCH_LESSER_EQUAL_1_10_2 = compare_version("torch", operator.le, "1.10.2")
 # duplicated from lite because HPU is patching it below
 _TORCH_GREATER_EQUAL_1_13 = compare_version("torch", operator.ge, "1.13.0")
+_TORCHMETRICS_GREATER_EQUAL_0_11 = compare_version("torchmetrics", operator.ge, "0.11.0")  # using new API with task
```

Comment:
This should be defined somewhere in tests/tests_pytorch since it's not used in the source code

Reply:
let's leave it here for now as TM is a mandatory dependency of PL